### DISCUSSION TITLE
Project teams

### DISCUSSION THREAD

**Discussion by jdmar3** [comp426-2023-spring/syllabus#21] [Announcements]
It has taken me all week but I am _finally_ starting to put the project teams into GitHub. I am including some other info along with and will notify you all when that is up. Most likely I will not be finished with that until tomorrow (Friday) sometime. So, enjoy the weekend and we can look at it when we come back next Thursday.

**Comment by sokhnarfall** [comment]
Hello, hope all is well!
I have been added to a team, but the individuals I asked to work with have not been added. Would you be able to add them since we all listed each other's names on the first exam? Their names and GitHub usernames are as follows; Richard Okoro- @ricokoro, Kirsten Allen- @amayak25. Thank you!

**Comment by Agaresd47** [comment]
Dear professor Martin,

My teammates and I were not on the same team though my teammate Zeqi ZHou previously emailed you. He will forward you again the team member as proof, and I also created another team: https://github.com/orgs/comp426-2023-spring/teams/team-recipe-rating

If you think that we can be a team, we will quit our previous teams. 

Our teammates are: 
Zeqi Zhou: JoeZhou-encoding
Zhuoyu Shi: zhuoyu73 (For some reason she is not available in the team to invite)
Boyu Liu: BoyuL129

Thus, we hope to use the team name: Team Recipe Rating to participate in the midterm and submit the final project as well. 

We work as above after your consent.

**Reply by zhuoyu73** [reply]
Hi Professor Martin, 

For some reason, I don't have the "team" page in the comp426 repo, and I was not assigned to any team. I joined COMP426 during the first week of class and finished the preliminary exam on Jan. 17. Could you please help me check whether I was added to the GitHub classroom? Thanks for your help! 
<img width="1160" alt="Screen Shot 2023-03-03 at 17 21 00" src="https://user-images.githubusercontent.com/97497879/222841511-03c7808c-910c-4e6d-b629-65b3940c04c3.png">

**Reply by AkshayW162** [reply]
Hi Professor, 
I am having the same issue as @zhuoyu73 where I cannot see the teams page in the repository. I was enrolled in this class prior to the start of the semester, however, so I am unsure why this is the case. Could I please get a follow-up on how to access the teams page so that I may coordinate with my team members and fill out the midterm?

**Comment by AkshayW162** [comment]
Hi, professor,
I have not received a response to this request: https://github.com/orgs/comp426-2023-spring/teams/ainternationalchat/discussions/1/comments/1
and I was hoping I could be added to the team before the midterm is due.
